packages/formation-react/src/components/ErrorableCheckbox/ErrorableCheckbox.jsx

```diff
@@ -88,13 +89,56 @@ ErrorableCheckbox.propTypes = {
    */
   name: PropTypes.string,
   /**
-   * Label for the checkbox
+   * Label [string or object] for the checkbox. Either this or ariaLabelledBy is required.
+   * IF you have a SEPARATE HEADING (outside this component) that “labels” this
+   * checkbox, use ariaLabelledBy instead of this.
+   * Screen-readers read sr-only content, which would be redundant if you had a
```

Comment:
I would remove these lines about `sr-only` content and the example. If we feel like that's necessary info, I think maybe putting it somewhere in the mdx page for this component would make more sense. It's going to hard to read in the prop types table.

Reply:
Done.